source/administration/encryption.rst

```diff
@@ -0,0 +1,34 @@
+Encryption Options
+=======================
+
+Mattermost provides encryption-in-transit and encryption-at-rest for both database and file storage. This page guides you through setting up appropriate encryption security.
+
+Encryption is not required for GDPR, although it can be used as an additional safeguard against data breach.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+Encryption-in-transit
+-----------------------
+
+Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet. You may either set up TLS on the Mattermost Server or install a proxy such as NGINX and set up TLS on the proxy. Refer to our `configuration guide for more details <https://docs.mattermost.com/install/config-tls-mattermost.html>`_.
```

Comment:
TLS supports many cipher suites: https://en.wikipedia.org/wiki/Transport_Layer_Security#Algorithm
Seems a bit silly to specify one. 

"across both LAN and internet". Is unnecessary specificity.